### PR TITLE
VS-1461 Instructions to create vid to mapping table

### PR DESCRIPTION
**Pull request by gbggrant**
Instructions to create vid to mapping table

### CODE REVIEW

File: scripts/variantstore/docs/aou/AOU_DELIVERABLES.md [hunk 1]
```diff
@@ -150,3 +150,47 @@ You can take advantage of our existing sub-cohort WDL, `GvsExtractCohortFromSamp
...
+    GROUP BY vat.vid
+    ORDER BY
+        vidToLocation(vat.vid),
+        SPLIT(vat.vid, '-')[OFFSET(2)],
+        SPLIT(vat.vid, '-')[OFFSET(3)]
+   ```
+1. Once the query has successfully finished, you should cluster it on the field `vid`. This can be accomplished using the command below. Note that you will need to specify the `project`, `dataset`, and `mapping_table_name` fields before running the command :
+```
+    bq update --project_id=<project> --clustering_fields=vid <dataset>.<mapping_table_name>
+```
```
Comment (rsasch) [inline scripts/variantstore/docs/aou/AOU_DELIVERABLES.md:194]: nit: this code block is less indented than the one above; they should probably have the same level of indentation

**Review by rsasch — APPROVED** [review]
one small nit, otherwise 👍🏻

**Review by RoriCremer — APPROVED** [review]

**Comment by RoriCremer** [comment]
would it be worth adding a note about how the mappings dont...map 100% ?

**Comment by mcovarr** [comment]
> would it be worth adding a note about how the mappings dont...map 100% ?

As I was not present at the mobbing for this and have no idea what this means, I would be in favor of adding the note. 🙂

**Review by mcovarr — APPROVED** [review]
two small nits otherwise 
I'm the map!

**Pull request merged**